toolkits/global/packages/global-forms/package.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "@springernature/global-forms",
-	"version": "4.0.1",
+	"version": "5.0.0",
 	"license": "MIT",
 	"description": "form component",
 	"keywords": [
 	  "css",
 	  "component"
 	],
   "author": "Jack Watkins",
-  "brandContext": "^18.0.0"
-  }
+  "brandContext": "^24.0.0"
```

Comment:
Should not this be 25 rather?

Reply:
@sturobson  Just done so! https://github.com/springernature/frontend-toolkits/pull/743/commits/227a9228fda6bfcce9109db7464bd4fa268204ba